cli/src/cluster_query.rs

```diff
@@ -1044,6 +1066,20 @@ pub fn process_leader_schedule(
     }))
 }
 
+pub fn process_get_recent_priority_fees(
+    rpc_client: &RpcClient,
+    _config: &CliConfig,
+    accounts: &[Pubkey],
+) -> ProcessResult {
+    let mut ret = String::new();
+    if let Ok(fees) = rpc_client.get_recent_prioritization_fees(accounts) {
+        for (fee, account) in fees.iter().zip(accounts.iter()) {
```

Comment:
`fees` are a list of min prio fees from recent (up to 150) blocks, if to write lock `accounts`; according to [RPC method](https://solana.com/docs/rpc/http/getrecentprioritizationfees). Should not `zip` them. 

For CLI, wdyt to return something like: 
```
Recent min prioritization fee from transactinos that write lock [accounts]:
Max: [max(fees)]
Mean: [mean{fees)]
Min: [min(fees)]
// or even 95th, etc
```
With `-vv` to print entire `fees` list.

Reply:
ok. I see.